package.json

```diff
@@ -52,6 +52,7 @@
     ]
   },
   "dependencies": {
+    "lru-cache": "^10.1.0",
```

Comment:
This is only being used in account package and have been already added there any reason adding in core package?